environment.yml

```diff
@@ -3,15 +3,16 @@ channels:
   - conda-forge
 
 dependencies:
-  # Python
   - python >=3.6
-  # Required
   - pip
   - geopandas
   - matplotlib-base
   - psycopg2
-  - postgis
   - requests
   - ipyleaflet
```

Comment:
Unrelated: why do you have the `ipyleaflet` NB dependency here? Is it required for geoDB? Otherwise I suggest removing it.

Reply:
Same applies to `matplotlib` above.

Reply:
They're needed in the sample notebooks - and thus, also for testing, as the sample notebooks are run during CI.